src/Compilers/VisualBasic/Portable/Symbols/Metadata/PE/PEAssemblySymbol.vb

```diff
@@ -257,5 +260,16 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols.Metadata.PE
         Public Overrides Function GetMetadata() As AssemblyMetadata
             Return _assembly.GetNonDisposableMetadata()
         End Function
+
+        Friend Function GetCompilerFeatureRequiredDiagnosticInfo() As DiagnosticInfo
+            If _lazyCachedCompilerFeatureRequiredDiagnosticInfo Is Nothing Then
```

Comment:
>Is Nothing [](http://example.com/codeflow?start=64&length=10)

Shouldn't this compare with a sentinel value instead? Why tests are passing?

Reply:
They're not. There's a merge conflict, so the results are stale.